### ISSUE TITLE
Compilation failed

### ISSUE THREAD

**Issue by nderambure** [JayDDee/cpuminer-opt#105]
Hi !

Since the 3.8.1, the compilation failed on this error on a up to date Debian jessie (8) : 

```
...
Makefile:2496: recipe for target 'algo/blake/cpuminer-blake-hash-4way.o' failed
make[2]: *** [algo/blake/cpuminer-blake-hash-4way.o] Error 1
make[2]: *** Attente des tâches non terminées....
mv -f algo/argon2/ar2/.deps/cpuminer-opt.Tpo algo/argon2/ar2/.deps/cpuminer-opt.Po
mv -f algo/blake/.deps/cpuminer-sph_blake.Tpo algo/blake/.deps/cpuminer-sph_blake.Po
mv -f algo/argon2/ar2/.deps/cpuminer-ar2-scrypt-jane.Tpo algo/argon2/ar2/.deps/cpuminer-ar2-scrypt-jane.Po
make[2]: Leaving directory '/home/dway/cryptoflooz/cpuminer-3.8.3.3'
Makefile:5473: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/home/dway/cryptoflooz/cpuminer-3.8.3.3'
Makefile:866: recipe for target 'all' failed
make: *** [all] Error 2
strip: « cpuminer »: pas de tel fichier
```

The last version that compiled well is 3.8.0.1.
Maybe Debian Jessie is too old now ?

**Comment by JayDDee** [comment]
Yet another issue with insufficient information.

**Comment by nderambure** [comment]
Hi JayDDee, what would you like for more information ?
I've just git clone any version from 3.8.1, and ran ./build.sh.
Any version failed compiling with the same error as I've pasted above.
I'm sure what I could provide to help you ?
I use a fresh Debian 8, up to date, with nothing more than packages needed for compiling.
Please tell me.

**Comment by JayDDee** [comment]
That's a lot better. This looks like a duplicate of 
https://github.com/JayDDee/cpuminer-opt/issues/84
It seems to be a problem with certain SIMD intrinsic functions not being recocgnized by
the compiler in Deb8. I can't confirm this because you didn't provide the first, and most important,
errors.

**Comment by nderambure** [comment]
OK here is the entire compilation log (I though this was not necessary as the error is at the end). But it seems exactly the same error than the #84. So don't waste time on this, Debian 8 is obsolete anyway...

```
dway@akira:~/cryptoflooz/cpuminer-3.8.3.3$ ./build.sh
make: *** No rule to make target 'distclean'. Arrêt.
clean
configure.ac:15: installing './compile'
configure.ac:4: installing './config.guess'
configure.ac:4: installing './config.sub'
configure.ac:9: installing './install-sh'
configure.ac:9: installing './missing'
Makefile.am: installing './depcomp'
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-gnu
checking target system type... x86_64-unknown-linux-gnu
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether to enable maintainer-specific portions of Makefiles... no
checking for style of include used by make... GNU
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether gcc understands -c and -o together... yes
checking dependency style of gcc... gcc3
checking for gcc option to accept ISO C99... -std=gnu99
checking how to run the C preprocessor... gcc -std=gnu99 -E
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking whether gcc -std=gnu99 needs -traditional... no
checking dependency style of gcc -std=gnu99... gcc3
checking for ranlib... ranlib
checking for g++... g++
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
checking dependency style of g++... gcc3
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking sys/endian.h usability... no
checking sys/endian.h presence... no
checking for sys/endian.h... no
checking sys/param.h usability... yes
checking sys/param.h presence... yes
checking for sys/param.h... yes
checking syslog.h usability... yes
checking syslog.h presence... yes
checking for syslog.h... yes
checking for sys/sysctl.h... yes
checking whether be32dec is declared... no
checking whether le32dec is declared... no
checking whether be32enc is declared... no
checking whether le32enc is declared... no
checking whether le16dec is declared... no
checking whether le16enc is declared... no
checking for size_t... yes
checking for working alloca.h... yes
checking for alloca... yes
checking for getopt_long... yes
checking whether we can compile AVX code... yes
checking whether we can compile XOP code... yes
checking whether we can compile AVX2 code... yes
checking for json_loads in -ljansson... yes
checking for pthread_create in -lpthread... yes
checking whether __uint128_t is supported... yes
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating compat/Makefile
config.status: creating compat/jansson/Makefile
config.status: creating cpuminer-config.h
config.status: executing depfiles commands
make  all-recursive
make[1]: Entering directory '/home/dway/cryptoflooz/cpuminer-3.8.3.3'
Making all in compat
make[2]: Entering directory '/home/dway/cryptoflooz/cpuminer-3.8.3.3/compat'
make[3]: Entering directory '/home/dway/cryptoflooz/cpuminer-3.8.3.3/compat'
make[3]: Nothing to be done for 'all-am'.
make[3]: Leaving directory '/home/dway/cryptoflooz/cpuminer-3.8.3.3/compat'
make[2]: Leaving directory '/home/dway/cryptoflooz/cpuminer-3.8.3.3/compat'
make[2]: Entering directory '/home/dway/cryptoflooz/cpuminer-3.8.3.3'
gcc -std=gnu99 -DHAVE_CONFIG_H -I.  -Iyes/include -fno-strict-aliasing  -I. -Iyes/include -Wno-pointer-sign -Wno-pointer-to-int-cast   -O3 -march=native -Wall  -Iyes/include -MT cpuminer-cpu-miner.o -MD -MP -MF .deps/cpuminer-cpu-miner.Tpo -c -o cpuminer-cpu-miner.o `test -f 'cpu-miner.c' || echo './'`cpu-miner.c
gcc -std=gnu99 -DHAVE_CONFIG_H -I.  -Iyes/include -fno-strict-aliasing  -I. -Iyes/include -Wno-pointer-sign -Wno-pointer-to-int-cast   -O3 -march=native -Wall  -Iyes/include -MT cpuminer-util.o -MD -MP -MF .deps/cpuminer-util.Tpo -c -o cpuminer-util.o `test -f 'util.c' || echo './'`util.c
g++ -DHAVE_CONFIG_H -I.  -Iyes/include -fno-strict-aliasing  -I. -Iyes/include  -g -O2 -MT cpuminer-uint256.o -MD -MP -MF .deps/cpuminer-uint256.Tpo -c -o cpuminer-uint256.o `test -f 'uint256.cpp' || echo './'`uint256.cpp
gcc -std=gnu99 -DHAVE_CONFIG_H -I.  -Iyes/include -fno-strict-aliasing  -I. -Iyes/include -Wno-pointer-sign -Wno-pointer-to-int-cast   -O3 -march=native -Wall  -Iyes/include -MT cpuminer-api.o -MD -MP -MF .deps/cpuminer-api.Tpo -c -o cpuminer-api.o `test -f 'api.c' || echo './'`api.c
mv -f .deps/cpuminer-uint256.Tpo .deps/cpuminer-uint256.Po
gcc -std=gnu99 -DHAVE_CONFIG_H -I.  -Iyes/include -fno-strict-aliasing  -I. -Iyes/include -Wno-pointer-sign -Wno-pointer-to-int-cast   -O3 -march=native -Wall  -Iyes/include -MT cpuminer-sysinfos.o -MD -MP -MF .deps/cpuminer-sysinfos.Tpo -c -o cpuminer-sysinfos.o `test -f 'sysinfos.c' || echo './'`sysinfos.c
mv -f .deps/cpuminer-api.Tpo .deps/cpuminer-api.Po
gcc -std=gnu99 -DHAVE_CONFIG_H -I.  -Iyes/include -fno-strict-aliasing  -I. -Iyes/include -Wno-pointer-sign -Wno-pointer-to-int-cast   -O3 -march=native -Wall  -Iyes/include -MT cpuminer-algo-gate-api.o -MD -MP -MF .deps/cpuminer-algo-gate-api.Tpo -c -o cpuminer-algo-gate-api.o `test -f 'algo-gate-api.c' || echo './'`algo-gate-api.c
mv -f .deps/cpuminer-sysinfos.Tpo .deps/cpuminer-sysinfos.Po
mv -f .deps/cpuminer-algo-gate-api.Tpo .deps/cpuminer-algo-gate-api.Po
gcc -std=gnu99 -DHAVE_CONFIG_H -I.  -Iyes/include -fno-strict-aliasing  -I. -Iyes/include -Wno-pointer-sign -Wno-pointer-to-int-cast   -O3 -march=native -Wall  -Iyes/include -MT algo/cpuminer-pluck.o -MD -MP -MF algo/.deps/cpuminer-pluck.Tpo -c -o algo/cpuminer-pluck.o `test -f 'algo/pluck.c' || echo './'`algo/pluck.c
gcc -std=gnu99 -DHAVE_CONFIG_H -I.  -Iyes/include -fno-strict-aliasing  -I. -Iyes/include -Wno-pointer-sign -Wno-pointer-to-int-cast   -O3 -march=native -Wall  -Iyes/include -MT algo/cpuminer-scrypt.o -MD -MP -MF algo/.deps/cpuminer-scrypt.Tpo -c -o algo/cpuminer-scrypt.o `test -f 'algo/scrypt.c' || echo './'`algo/scrypt.c
mv -f .deps/cpuminer-util.Tpo .deps/cpuminer-util.Po
mv -f algo/.deps/cpuminer-pluck.Tpo algo/.deps/cpuminer-pluck.Po
gcc -std=gnu99 -DHAVE_CONFIG_H -I.  -Iyes/include -fno-strict-aliasing  -I. -Iyes/include -Wno-pointer-sign -Wno-pointer-to-int-cast   -O3 -march=native -Wall  -Iyes/include -MT crypto/cpuminer-oaes_lib.o -MD -MP -MF crypto/.deps/cpuminer-oaes_lib.Tpo -c -o crypto/cpuminer-oaes_lib.o `test -f 'crypto/oaes_lib.c' || echo './'`crypto/oaes_lib.c
gcc -std=gnu99 -DHAVE_CONFIG_H -I.  -Iyes/include -fno-strict-aliasing  -I. -Iyes/include -Wno-pointer-sign -Wno-pointer-to-int-cast   -O3 -march=native -Wall  -Iyes/include -MT crypto/cpuminer-c_keccak.o -MD -MP -MF crypto/.deps/cpuminer-c_keccak.Tpo -c -o crypto/cpuminer-c_keccak.o `test -f 'crypto/c_keccak.c' || echo './'`crypto/c_keccak.c
mv -f crypto/.deps/cpuminer-c_keccak.Tpo crypto/.deps/cpuminer-c_keccak.Po
gcc -std=gnu99 -DHAVE_CONFIG_H -I.  -Iyes/include -fno-strict-aliasing  -I. -Iyes/include -Wno-pointer-sign -Wno-pointer-to-int-cast   -O3 -march=native -Wall  -Iyes/include -MT crypto/cpuminer-c_groestl.o -MD -MP -MF crypto/.deps/cpuminer-c_groestl.Tpo -c -o crypto/cpuminer-c_groestl.o `test -f 'crypto/c_groestl.c' || echo './'`crypto/c_groestl.c
mv -f crypto/.deps/cpuminer-oaes_lib.Tpo crypto/.deps/cpuminer-oaes_lib.Po
gcc -std=gnu99 -DHAVE_CONFIG_H -I.  -Iyes/include -fno-strict-aliasing  -I. -Iyes/include -Wno-pointer-sign -Wno-pointer-to-int-cast   -O3 -march=native -Wall  -Iyes/include -MT crypto/cpuminer-c_blake256.o -MD -MP -MF crypto/.deps/cpuminer-c_blake256.Tpo -c -o crypto/cpuminer-c_blake256.o `test -f 'crypto/c_blake256.c' || echo './'`crypto/c_blake256.c
mv -f .deps/cpuminer-cpu-miner.Tpo .deps/cpuminer-cpu-miner.Po
gcc -std=gnu99 -DHAVE_CONFIG_H -I.  -Iyes/include -fno-strict-aliasing  -I. -Iyes/include -Wno-pointer-sign -Wno-pointer-to-int-cast   -O3 -march=native -Wall  -Iyes/include -MT crypto/cpuminer-c_jh.o -MD -MP -MF crypto/.deps/cpuminer-c_jh.Tpo -c -o crypto/cpuminer-c_jh.o `test -f 'crypto/c_jh.c' || echo './'`crypto/c_jh.c
mv -f crypto/.deps/cpuminer-c_groestl.Tpo crypto/.deps/cpuminer-c_groestl.Po
gcc -std=gnu99 -DHAVE_CONFIG_H -I.  -Iyes/include -fno-strict-aliasing  -I. -Iyes/include -Wno-pointer-sign -Wno-pointer-to-int-cast   -O3 -march=native -Wall  -Iyes/include -MT crypto/cpuminer-c_skein.o -MD -MP -MF crypto/.deps/cpuminer-c_skein.Tpo -c -o crypto/cpuminer-c_skein.o `test -f 'crypto/c_skein.c' || echo './'`crypto/c_skein.c
mv -f algo/.deps/cpuminer-scrypt.Tpo algo/.deps/cpuminer-scrypt.Po
gcc -std=gnu99 -DHAVE_CONFIG_H -I.  -Iyes/include -fno-strict-aliasing  -I. -Iyes/include -Wno-pointer-sign -Wno-pointer-to-int-cast   -O3 -march=native -Wall  -Iyes/include -MT crypto/cpuminer-hash.o -MD -MP -MF crypto/.deps/cpuminer-hash.Tpo -c -o crypto/cpuminer-hash.o `test -f 'crypto/hash.c' || echo './'`crypto/hash.c
mv -f crypto/.deps/cpuminer-hash.Tpo crypto/.deps/cpuminer-hash.Po
gcc -std=gnu99 -DHAVE_CONFIG_H -I.  -Iyes/include -fno-strict-aliasing  -I. -Iyes/include -Wno-pointer-sign -Wno-pointer-to-int-cast   -O3 -march=native -Wall  -Iyes/include -MT crypto/cpuminer-aesb.o -MD -MP -MF crypto/.deps/cpuminer-aesb.Tpo -c -o crypto/cpuminer-aesb.o `test -f 'crypto/aesb.c' || echo './'`crypto/aesb.c
mv -f crypto/.deps/cpuminer-c_jh.Tpo crypto/.deps/cpuminer-c_jh.Po
g++ -DHAVE_CONFIG_H -I.  -Iyes/include -fno-strict-aliasing  -I. -Iyes/include  -g -O2 -MT crypto/cpuminer-magimath.o -MD -MP -MF crypto/.deps/cpuminer-magimath.Tpo -c -o crypto/cpuminer-magimath.o `test -f 'crypto/magimath.cpp' || echo './'`crypto/magimath.cpp
mv -f crypto/.deps/cpuminer-aesb.Tpo crypto/.deps/cpuminer-aesb.Po
gcc -std=gnu99 -DHAVE_CONFIG_H -I.  -Iyes/include -fno-strict-aliasing  -I. -Iyes/include -Wno-pointer-sign -Wno-pointer-to-int-cast   -O3 -march=native -Wall  -Iyes/include -MT algo/argon2/cpuminer-argon2a.o -MD -MP -MF algo/argon2/.deps/cpuminer-argon2a.Tpo -c -o algo/argon2/cpuminer-argon2a.o `test -f 'algo/argon2/argon2a.c' || echo './'`algo/argon2/argon2a.c
mv -f algo/argon2/.deps/cpuminer-argon2a.Tpo algo/argon2/.deps/cpuminer-argon2a.Po
gcc -std=gnu99 -DHAVE_CONFIG_H -I.  -Iyes/include -fno-strict-aliasing  -I. -Iyes/include -Wno-pointer-sign -Wno-pointer-to-int-cast   -O3 -march=native -Wall  -Iyes/include -MT algo/argon2/ar2/cpuminer-argon2.o -MD -MP -MF algo/argon2/ar2/.deps/cpuminer-argon2.Tpo -c -o algo/argon2/ar2/cpuminer-argon2.o `test -f 'algo/argon2/ar2/argon2.c' || echo './'`algo/argon2/ar2/argon2.c
mv -f algo/argon2/ar2/.deps/cpuminer-argon2.Tpo algo/argon2/ar2/.deps/cpuminer-argon2.Po
gcc -std=gnu99 -DHAVE_CONFIG_H -I.  -Iyes/include -fno-strict-aliasing  -I. -Iyes/include -Wno-pointer-sign -Wno-pointer-to-int-cast   -O3 -march=native -Wall  -Iyes/include -MT algo/argon2/ar2/cpuminer-opt.o -MD -MP -MF algo/argon2/ar2/.deps/cpuminer-opt.Tpo -c -o algo/argon2/ar2/cpuminer-opt.o `test -f 'algo/argon2/ar2/opt.c' || echo './'`algo/argon2/ar2/opt.c
mv -f crypto/.deps/cpuminer-c_blake256.Tpo crypto/.deps/cpuminer-c_blake256.Po
gcc -std=gnu99 -DHAVE_CONFIG_H -I.  -Iyes/include -fno-strict-aliasing  -I. -Iyes/include -Wno-pointer-sign -Wno-pointer-to-int-cast   -O3 -march=native -Wall  -Iyes/include -MT algo/argon2/ar2/cpuminer-cores.o -MD -MP -MF algo/argon2/ar2/.deps/cpuminer-cores.Tpo -c -o algo/argon2/ar2/cpuminer-cores.o `test -f 'algo/argon2/ar2/cores.c' || echo './'`algo/argon2/ar2/cores.c
mv -f crypto/.deps/cpuminer-magimath.Tpo crypto/.deps/cpuminer-magimath.Po
gcc -std=gnu99 -DHAVE_CONFIG_H -I.  -Iyes/include -fno-strict-aliasing  -I. -Iyes/include -Wno-pointer-sign -Wno-pointer-to-int-cast   -O3 -march=native -Wall  -Iyes/include -MT algo/argon2/ar2/cpuminer-ar2-scrypt-jane.o -MD -MP -MF algo/argon2/ar2/.deps/cpuminer-ar2-scrypt-jane.Tpo -c -o algo/argon2/ar2/cpuminer-ar2-scrypt-jane.o `test -f 'algo/argon2/ar2/ar2-scrypt-jane.c' || echo './'`algo/argon2/ar2/ar2-scrypt-jane.c
In file included from algo/argon2/ar2/sj/scrypt-jane-salsa64.h:12:0,
                 from algo/argon2/ar2/sj/scrypt-jane-romix.h:2,
                 from algo/argon2/ar2/ar2-scrypt-jane.c:19:
algo/argon2/ar2/sj/scrypt-jane-romix-basic.h:8:1: warning: ‘scrypt_romix_nop’ defined but not used [-Wunused-function]
 scrypt_romix_nop(scrypt_mix_word_t *blocks, size_t nblocks) {
 ^
mv -f algo/argon2/ar2/.deps/cpuminer-cores.Tpo algo/argon2/ar2/.deps/cpuminer-cores.Po
gcc -std=gnu99 -DHAVE_CONFIG_H -I.  -Iyes/include -fno-strict-aliasing  -I. -Iyes/include -Wno-pointer-sign -Wno-pointer-to-int-cast   -O3 -march=native -Wall  -Iyes/include -MT algo/argon2/ar2/cpuminer-blake2b.o -MD -MP -MF algo/argon2/ar2/.deps/cpuminer-blake2b.Tpo -c -o algo/argon2/ar2/cpuminer-blake2b.o `test -f 'algo/argon2/ar2/blake2b.c' || echo './'`algo/argon2/ar2/blake2b.c
mv -f algo/argon2/ar2/.deps/cpuminer-blake2b.Tpo algo/argon2/ar2/.deps/cpuminer-blake2b.Po
gcc -std=gnu99 -DHAVE_CONFIG_H -I.  -Iyes/include -fno-strict-aliasing  -I. -Iyes/include -Wno-pointer-sign -Wno-pointer-to-int-cast   -O3 -march=native -Wall  -Iyes/include -MT algo/blake/cpuminer-sph_blake.o -MD -MP -MF algo/blake/.deps/cpuminer-sph_blake.Tpo -c -o algo/blake/cpuminer-sph_blake.o `test -f 'algo/blake/sph_blake.c' || echo './'`algo/blake/sph_blake.c
mv -f crypto/.deps/cpuminer-c_skein.Tpo crypto/.deps/cpuminer-c_skein.Po
gcc -std=gnu99 -DHAVE_CONFIG_H -I.  -Iyes/include -fno-strict-aliasing  -I. -Iyes/include -Wno-pointer-sign -Wno-pointer-to-int-cast   -O3 -march=native -Wall  -Iyes/include -MT algo/blake/cpuminer-blake-hash-4way.o -MD -MP -MF algo/blake/.deps/cpuminer-blake-hash-4way.Tpo -c -o algo/blake/cpuminer-blake-hash-4way.o `test -f 'algo/blake/blake-hash-4way.c' || echo './'`algo/blake/blake-hash-4way.c
In file included from algo/blake/blake-hash-4way.h:48:0,
                 from algo/blake/blake-hash-4way.c:39:
./avxdefs.h: In function ‘mm_brotr_128’:
./avxdefs.h:459:3: warning: implicit declaration of function ‘_mm_bsrli_si128’ [-Wimplicit-function-declaration]
   return _mm_or_si128( _mm_bsrli_si128( v, c ), _mm_bslli_si128( v, 16-(c) ) );}
   ^
./avxdefs.h:459:3: warning: implicit declaration of function ‘_mm_bslli_si128’ [-Wimplicit-function-declaration]
./avxdefs.h:459:10: error: incompatible type for argument 1 of ‘_mm_or_si128’
   return _mm_or_si128( _mm_bsrli_si128( v, c ), _mm_bslli_si128( v, 16-(c) ) );}
          ^
In file included from /usr/lib/gcc/x86_64-linux-gnu/4.9/include/xmmintrin.h:1258:0,
                 from /usr/lib/gcc/x86_64-linux-gnu/4.9/include/immintrin.h:29,
                 from ./avxdefs.h:42,
                 from algo/blake/blake-hash-4way.h:48,
                 from algo/blake/blake-hash-4way.c:39:
/usr/lib/gcc/x86_64-linux-gnu/4.9/include/emmintrin.h:1260:1: note: expected ‘__m128i’ but argument is of type ‘int’
 _mm_or_si128 (__m128i __A, __m128i __B)
 ^
In file included from algo/blake/blake-hash-4way.h:48:0,
                 from algo/blake/blake-hash-4way.c:39:
./avxdefs.h:459:10: error: incompatible type for argument 2 of ‘_mm_or_si128’
   return _mm_or_si128( _mm_bsrli_si128( v, c ), _mm_bslli_si128( v, 16-(c) ) );}
          ^
In file included from /usr/lib/gcc/x86_64-linux-gnu/4.9/include/xmmintrin.h:1258:0,
                 from /usr/lib/gcc/x86_64-linux-gnu/4.9/include/immintrin.h:29,
                 from ./avxdefs.h:42,
                 from algo/blake/blake-hash-4way.h:48,
                 from algo/blake/blake-hash-4way.c:39:
/usr/lib/gcc/x86_64-linux-gnu/4.9/include/emmintrin.h:1260:1: note: expected ‘__m128i’ but argument is of type ‘int’
 _mm_or_si128 (__m128i __A, __m128i __B)
 ^
In file included from algo/blake/blake-hash-4way.h:48:0,
                 from algo/blake/blake-hash-4way.c:39:
./avxdefs.h: In function ‘mm_brotl_128’:
./avxdefs.h:463:10: error: incompatible type for argument 1 of ‘_mm_or_si128’
   return _mm_or_si128( _mm_bslli_si128( v, c ), _mm_bsrli_si128( v, 16-(c) ) );
          ^
In file included from /usr/lib/gcc/x86_64-linux-gnu/4.9/include/xmmintrin.h:1258:0,
                 from /usr/lib/gcc/x86_64-linux-gnu/4.9/include/immintrin.h:29,
                 from ./avxdefs.h:42,
                 from algo/blake/blake-hash-4way.h:48,
                 from algo/blake/blake-hash-4way.c:39:
/usr/lib/gcc/x86_64-linux-gnu/4.9/include/emmintrin.h:1260:1: note: expected ‘__m128i’ but argument is of type ‘int’
 _mm_or_si128 (__m128i __A, __m128i __B)
 ^
In file included from algo/blake/blake-hash-4way.h:48:0,
                 from algo/blake/blake-hash-4way.c:39:
./avxdefs.h:463:10: error: incompatible type for argument 2 of ‘_mm_or_si128’
   return _mm_or_si128( _mm_bslli_si128( v, c ), _mm_bsrli_si128( v, 16-(c) ) );
          ^
In file included from /usr/lib/gcc/x86_64-linux-gnu/4.9/include/xmmintrin.h:1258:0,
                 from /usr/lib/gcc/x86_64-linux-gnu/4.9/include/immintrin.h:29,
                 from ./avxdefs.h:42,
                 from algo/blake/blake-hash-4way.h:48,
                 from algo/blake/blake-hash-4way.c:39:
/usr/lib/gcc/x86_64-linux-gnu/4.9/include/emmintrin.h:1260:1: note: expected ‘__m128i’ but argument is of type ‘int’
 _mm_or_si128 (__m128i __A, __m128i __B)
 ^
Makefile:2496: recipe for target 'algo/blake/cpuminer-blake-hash-4way.o' failed
make[2]: *** [algo/blake/cpuminer-blake-hash-4way.o] Error 1
make[2]: *** Attente des tâches non terminées....
mv -f algo/argon2/ar2/.deps/cpuminer-opt.Tpo algo/argon2/ar2/.deps/cpuminer-opt.Po
mv -f algo/argon2/ar2/.deps/cpuminer-ar2-scrypt-jane.Tpo algo/argon2/ar2/.deps/cpuminer-ar2-scrypt-jane.Po
mv -f algo/blake/.deps/cpuminer-sph_blake.Tpo algo/blake/.deps/cpuminer-sph_blake.Po
make[2]: Leaving directory '/home/dway/cryptoflooz/cpuminer-3.8.3.3'
Makefile:5473: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/home/dway/cryptoflooz/cpuminer-3.8.3.3'
Makefile:866: recipe for target 'all' failed
make: *** [all] Error 2
strip: « cpuminer »: pas de tel fichier
```

**Comment by JayDDee** [comment]
This is the first error and the most important one. It confirms it is the same problem as previously
reported. As this is a problem within Debian 8 and is not present in the newer Debian 9 and any fix
would have a performance impact, no attempt will be made to fix it. However it wil be added to the
errata.

In file included from algo/blake/blake-hash-4way.h:48:0,
                 from algo/blake/blake-hash-4way.c:39:
./avxdefs.h: In function ‘mm_brotr_128’:
./avxdefs.h:459:3: warning: implicit declaration of function ‘_mm_bsrli_si128’ [-Wimplicit-function-declaration]
   return _mm_or_si128( _mm_bsrli_si128( v, c ), _mm_bslli_si128( v, 16-(c) ) );}
   ^
./avxdefs.h:459:3: warning: implicit declaration of function ‘_mm_bslli_si128’ [-Wimplicit-function-declaration]
./avxdefs.h:459:10: error: incompatible type for argument 1 of ‘_mm_or_si128’
   return _mm_or_si128( _mm_bsrli_si128( v, c ), _mm_bslli_si128( v, 16-(c) ) );}
          ^
